### PR TITLE
chore: update the links for ephemeral-cluster example demo and docs

### PR DESCRIPTION
**Pull request by shumailxyz**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/basics/ephemeral-clusters.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Garden will automatically provision an Ephemeral Kubernetes Cluster for your pro
 
 Ephemeral Kubernetes Clusters fully support ingresses and each cluster is assigned its own unique default hostname dynamically when created. This hostname and its direct subdomains are secured by TLS and require authentication.
 
-### Configuring ingress
+### Configuring the ingress
```
Comment (twelvemo) [inline docs/basics/ephemeral-clusters.md:51]: I actually think configuring ingress is a bit better here, because it does not specify it is only one.

**Review by twelvemo — CHANGES_REQUESTED** [review]

File: docs/basics/ephemeral-clusters.md (hunk 1 above)
Comment (shumailxyz) [reply docs/basics/ephemeral-clusters.md:51]: I was reading it multiple times and was not sure about it actually

Comment (shumailxyz) [reply docs/basics/ephemeral-clusters.md]: removed

File: docs/basics/ephemeral-clusters.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ ingresses:
       hostname: api.${providers.ephemeral-kubernetes.outputs.default-hostname}
 ```
 
-### Authentication for ingress
+### Authentication for the ingress
```
Comment (twelvemo) [inline docs/basics/ephemeral-clusters.md:65]: sorry to be a terrible nit-pick, but can we remove it here too?
```suggestion
### Authentication for ingress
```

**Review by twelvemo — APPROVED** [review]

File: docs/basics/ephemeral-clusters.md (hunk 2 above)
Comment (shumailxyz) [reply docs/basics/ephemeral-clusters.md]: no worries, improvements are always welcome :)

**Pull request merged**